openedx/core/djangoapps/profile_images/images.py

```diff
@@ -0,0 +1,128 @@
+"""
+Image file manipulation functions related to profile images.
+"""
+from cStringIO import StringIO
+
+from django.conf import settings
+from django.core.files.base import ContentFile
+from PIL import Image
+
+from ..user_api.accounts.api import get_profile_image_storage
+
+
+class DevMsg(object):
+    """
+    Holder for pseudo-constants.
+    """
+    FILE_TOO_LARGE = 'Maximum file size exceeded.'
```

Comment:
Right, but should these messages be passed as the user_message as well. Are these exactly the sort of messages we want to return to the client?